config/__tests__/config.ts

```diff
@@ -103,6 +105,16 @@ function getAccountByAuthType(
 }
 
 describe('config/config', () => {
+
+  const globalConsole = global.console;
+  beforeAll(() => {
+    global.console.error = jest.fn();
+    global.console.debug = jest.fn();
```

Comment:
Some of the methods in `config_DEPRECATED` are directly calling `console.error` and `console.debug`